### PR TITLE
server: use seconds instead of nano seconds for expires_in and expiry

### PR DESCRIPTION
**Pull request by ericchiang**

### CODE REVIEW

File: server/server_test.go [hunk 1]
```diff
@@ -137,6 +137,14 @@ func TestOAuth2CodeFlow(t *testing.T) {
 	clientSecret := "testclientsecret"
 	requestedScopes := []string{oidc.ScopeOpenID, "email", "offline_access"}
 
+	t0 := time.Now().Round(time.Second)
+
+	// Always have the time function used by the server return the same time so
+	// we can predict expected values of "expires_in" fields exactly.
+	now := func() time.Time { return t0 }
+
+	idTokensValidFor := time.Second * 30
```
Comment (rithujohn191) [inline server/server_test.go]: Maybe a small comment explaining why it is initialized to this value would help.

File: server/server_test.go [hunk 2]
```diff
@@ -137,6 +137,18 @@ func TestOAuth2CodeFlow(t *testing.T) {
 	clientSecret := "testclientsecret"
 	requestedScopes := []string{oidc.ScopeOpenID, "email", "offline_access"}
 
+	t0 := time.Now().Round(time.Second)
+
+	// Always have the time function used by the server return the same time so
+	// we can predict expected values of "expires_in" fields exactly.
+	now := func() time.Time { return t0 }
+
+	// Used later when configuring test servers to set how long id_tokens will be valid for.
```
Comment (ericchiang) [inline server/server_test.go]: @rithujohn191 does this clarify this field?

Comment (rithujohn191) [inline server/server_test.go]: perfect

**Comment by rithujohn191** [comment]
lgtm.

**Pull request merged**